src/Explorer/Panes/AddCollectionPanel.tsx

```diff
@@ -833,12 +909,20 @@ export class AddCollectionPanel extends React.Component<AddCollectionPanelProps,
     return isLowerCase ? partitionKeyName.toLocaleLowerCase() : partitionKeyName;
   }
 
-  private getPartitionKeyPlaceHolder(): string {
+  private getPartitionKeyPlaceHolder(index?: number): string {
     switch (userContext.apiType) {
       case "Mongo":
         return "e.g., address.zipCode";
       case "Gremlin":
         return "e.g., /address";
+      case "SQL":
```

Comment:
Can we combine the two conditional operators into one? It looks like the conditions are the same (index = 0, 1, and 2).